src/app/public/modules/lookup/lookup.component.ts

```diff
@@ -197,17 +202,34 @@ export class SkyLookupComponent
   }
 
   public set tokens(value: SkyToken[]) {
-    this._tokens = value;
-    this.onChange(this.value);
-    this.onTouched();
+    // Collapse the tokens into a single token if the user has selected many options.
+    if (this.enableShowMore && this.value.length > 5) {
+      this.resourcesService.getString(
+        'skyux_lookup_tokens_summary',
+        this.value.length.toString()
+      )
+        .pipe(take(1))
+        .subscribe((label) => {
+          this._tokens = [{
+            value: { [this.descriptorProperty]: label }
+          }];
+          this.changeDetector.markForCheck();
+        });
+    } else {
+      this._tokens = value;
```

Comment:
Should we follow this branch with a `this.changeDetector.markForCheck();` as well?